src/components/Routes/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {NavLink} from "react-router-dom";
+import styles from './styles.module.sass'
+
+const activeStyle = {
+  backgroundColor : 'hsla(216, 85%, 34%,1)',
+  fontWeight:'900'
+}
+const Routes = ()=>{
```

Comment:
Bad component name. This is the visual representation of the routes in the shape of a navigation menu not the actual routes. use `Nav` or `Navigation` instead